project-codeflare/codeflare-sdk
# Added docs for Config and Auth & update doc workflow

# Issue link


Closes #429
# What changes have been made

The detailed "code orientated" docs have been moved to a new folder within docs called `detailed-docs`
The release workflow has been updated to account for this.
New docs were added for Cluster Configuration and Authentication

Ingress instructions in README moved to cluster-configuration.md
# Verification steps



You can verify the workflow changes by running this command.
```
poetry run pdoc --html -o docs/detailed-documentation src/codeflare_sdk && pushd docs/detailed-documentation && rm -rf cluster job utils && mv codeflare_sdk/* . && rm -rf codeflare_sdk && popd && find docs/detailed-documentation -type f -name "*.html" -exec bash -c "echo '' >> {}" \;
``` 
You should get a docs/detailed-docs folder containing the detailed documentation.
## Checks
- Testing Strategy
